spec/controllers/admin/works_controller_spec.rb

```diff
@@ -321,7 +322,8 @@
       let(:work) do
         create(:work, :published).tap do |w|
           w.members += [asset_child, work_child]
-          w.published = false
+          w.published = false          
+          w.published_at = nil
```

Comment:
Normally we can just do:

    create(:work, :published, published_at: nil)

I think the members/published stuff is being done in a `tap` because of weird previous stuff around representative validation or something? Not sure why else we do `create(:work, :published)`, and then immediately set `published` to false! We don't need to fix that problem here (although we could), but let's not add to the weirdness with more in the `tap` if we don't need to... but really not sure what's going on this test altogether, it's a weird test where we create a published work and then set it to not published and then... publish it?

Actually... the logic you added is in the _model_, we should maybe test it in a model test rather than a controller test? Maybe leave the controller test alone, but add a test in a work model test?

Reply:
Yeah. I'm adding a model test, and I can totally fix that `tap`. It doesn't need to be there anymore- correct.